src/cmd/run.go

```diff
@@ -78,7 +78,10 @@ func doRun(cmd *cobra.Command, args []string) {
 		log.Info().Msgf("interupt - waiting for jobs to complete ...")
 		wg.Wait()
 		log.Info().Msgf("Unregister runner for id '%s'...", runner.Id)
-		client.RunnerUnregister(runner.Id)
+		err = client.RunnerUnregister(runner.Id)
+		if err != nil {
+			log.Error().Err(err).Msgf("received error while unregistering runner")
```

Comment:
can we actually throw a message to sentry at this point?

Reply:
@rocktavious sounds like a great idea, I've made a ticket to add that to next sprint. https://github.com/OpsLevel/team-platform/issues/273